webapp/src/Controller/API/ContestController.php

```diff
@@ -237,10 +238,6 @@ public function bannerAction(Request $request, string $id): Response
      *     description="Contest start time changed successfully",
      * )
      * @OA\Response(
-     *     response="400",
-     *     description="Invalid input data"
```

Comment:
@nickygerritsen this one.

Reply:
Yeah I don't care about that one at all 😄